### DISCUSSION TITLE
WAZUH - CIS Benchmarks

### DISCUSSION THREAD

**Discussion by mrocaruano** [Security-Onion-Solutions/securityonion#5677] [Unsupported Versions] [answered]
Is there an estimate on when Wazuh will be upgraded? I am working on implementing Wazuh to monitor CIS Benchmark tests against various systems in our organization. As I was reading the Wazuh Documentation, I noticed that the cis_deian9 yml file is for CIS Benchmark for Debian/Linux 9 which is Intended for Debian 9/ Ubuntu 16. the cis_debian10 file is not supported yet.

![image](https://user-images.githubusercontent.com/43893495/134984128-cb1b3a74-7e09-4196-a8ce-4a48b5db0879.png)

 This is outdated. Is there anyway that benchmark files can be updated in the next release of security onion, or is there a quick fix/documentation on how to add custom benchmarks to be a applied to all Wazuh Agents? 

I found a solution on implementing a custom benchmark but it required uploading and changing settings on each individual agent, I want to be able to upload one yml file to the manager wazuh agent and be applied to all wazuh agents on start time. Any information would help, thank you!

**Comment by dougburks** [comment, marked as answer]
You can track the progress of the Wazuh upgrade via https://github.com/Security-Onion-Solutions/securityonion/issues/1748. 

We have no estimate right now, but will take a look as time allows.
